core/src/models/App.ts

```diff
@@ -207,6 +211,23 @@ export class App extends LoggedModel<App> {
     return { source, destination };
   }
 
+  async setConfigFilePath(newPath?: string) {
+    this.configFilePath = newPath ? newPath : `apps/${this.id}.json`;
+    await this.save();
+  }
```

Comment:
I went back and forth on whether I should abstract this elsewhere. In the end, it's so simple that I kept it on the instance and am testing it in ConfigWriter specs.